packages/buttons/src/styled/StyledExternalIcon.ts

```diff
@@ -14,7 +14,7 @@ const COMPONENT_ID = 'buttons.external_icon';
 /**
  * Accepts all `<svg>` props
  */
-export const StyledExternalIcon = styled(NewWindowIcon).attrs({
+export const StyledExternalIcon: any = styled(NewWindowIcon).attrs({
```

Comment:
Was there an issue with the typing to use `: any`?

Reply:
Yeah — VSCode wasn't happy that I hadn't explicitly declared the type here. That was when I had put the `useText` hook in here, though...maybe that'll be different, now that it's in the Anchor itself? Will investigate.

Reply:
Was able to remove it. 👍